### DISCUSSION TITLE
2024-10-13-intuition

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#190] [General]
# 别相信直觉 - 读书笔记 - Tw93

Hi, I'm Tw93, a Frontend Engineer from China, also dabbling a bit in Swift, Rust, Python, and Sketch. I'm the creator of MiaoYan, Pake, Trendy Weekly, XRender, and WeexUi. Believing in long-termism, I am passionate about open source and enjoy experimenting with new ideas.Share最近看完了「别相信直觉」用数据思维获取你想要的生活这本小书，比较简单好读，基本上花一下午就可以看完，把生活中可能会遇到的婚恋、育儿、财富、创业、运气、外貌、快乐、痛苦都给讲了讲，把里面有一些有意思的观点记录到这里。

https://tw93.fun/2024-10-13/intuition.html

**Comment by yexia553** [comment]
感觉很有趣的一本书

**Reply by tw93** [reply]
建议用电子书看，哈哈，这种小书不太适合买实体书

**Comment by Apecice** [comment]
快乐排名活动表 
是 ORDER BY ‘快乐’ DESC（按多巴胺的值倒序）吗？
